tmt-app/src/Components/Home.js

```diff
@@ -1,63 +1,112 @@
 import React from "react";
-import '../Styling/Home.css'
+import '../Styling/Home.css';
+import {PrimaryBtn, FlexContainer} from '../Styling/CustomStyling.js';
+import {Link, Container, Typography} from '@mui/material';
 import ForwardArrow from '../Images/Forward.png';
 import MilkTeaBlob from '../Images/MilkTeaBlob.png';
 import LycheeRectangle from '../Images/LycheeRectangle.png';
 import MilkTeaSwiggle2 from '../Images/MilkTeaSwiggle2.png';
 import ThaiTeaSwiggle1 from '../Images/ThaiTeaSwiggle1.png';
 import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
 
+
+
 function Home(){
     return(
-        <div className="Home_Container">
-
-            <div className="Home_Path_Container">
+        <Container 
+            sx={{
+                minWidth:"100%", 
+                paddingTop:"7.5rem", 
+                overflow:"hidden", 
+                minHeight:"100vh"
+            }}
+        >
+            <Container>
                 <div className="Home_Path_Wrapper">
-                    <span>Student Projects</span>
-                    <img src={ForwardArrow} alt="ForwardArrow"></img>
-                    <span>Major Picker</span>
+                    <span>
```

Comment:
Working with `Typography`, this can replace the need for `<span>` elements here and clean up lots of the markup we have